### PR TITLE
adding parse malware

### PR DESCRIPTION
**Pull request by maimorag**
## Status
- [x] In Progress
- [ ] Ready

## Related Issues
fixes:[ link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-11498)

## Description
Adding Parse Malware

### CODE REVIEW

**Review by yaakovpraisler — APPROVED** [review]

File: Packs/FeedUnit42v2/Integrations/FeedUnit42v2/FeedUnit42v2.py [hunk 1]
```diff
@@ -142,6 +142,27 @@ def parse_indicators(indicator_objects: list, feed_tags: Optional[list] = None,
...
+def parse_malware(client, malware_objects: list, feed_tags: Optional[list] = None,
+                  tlp_color: Optional[str] = None) -> list:
+    """Parse the IOC objects retrieved from the feed.
+    Args:
+      malware_objects: a list of objects containing the instances of malware.
+    Returns:
+        A list of processed malware.
+    """
+    if not feed_tags:
+        feed_tags = []
```
Comment (eyalpalo) [inline Packs/FeedUnit42v2/Integrations/FeedUnit42v2/FeedUnit42v2.py:154]: Why not setting default value to be an empty list in the method args?

File: Packs/FeedUnit42v2/Integrations/FeedUnit42v2/FeedUnit42v2.py [hunk 2]
```diff
@@ -142,6 +142,27 @@ def parse_indicators(indicator_objects: list, feed_tags: Optional[list] = None,
...
+    malware_list = []
+    if malware_objects:
+        for malware_object in malware_objects:
```
Comment (eyalpalo) [inline Packs/FeedUnit42v2/Integrations/FeedUnit42v2/FeedUnit42v2.py:157]: if the default value of malware_object will be empty list then we wont need the if. What are the possible values of malware_objects?

File: Packs/FeedUnit42v2/Integrations/FeedUnit42v2/test_data/feed_data.py [hunk 3]
```diff
@@ -867,6 +867,119 @@
      "name": "[Endor] Maze Ransomware"}
 ]
 
+INDICATOR_WITH_RELATIONSHIP = {
```
Comment (eyalpalo) [inline Packs/FeedUnit42v2/Integrations/FeedUnit42v2/test_data/feed_data.py:870]: Lets revert all changes that weren't made by you.

**Review by eyalpalo — CHANGES_REQUESTED** [review]
Nice job, lets talk tomorrow regarding this PR before merging

**Pull request merged**